py/LICENSE

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
@torsteingrindvik We need to choose a license

Reply:
Ah, could you copy the two license files from https://github.com/torsteingrindvik/bevy-vfx-bag ?
The license string can be found here: https://github.com/torsteingrindvik/bevy-vfx-bag/blob/main/Cargo.toml#L8